test/test_response.py

```diff
@@ -2,6 +2,7 @@
 
 from io import BytesIO, BufferedReader
 
+from httplib import HTTPResponse as HLHTTPResponse
```

Comment:
`httplib_HTTPResponse` might be clearer (had a few double-takes about what an HLHTTP is.) :)